TwilioVerifySDKTests/TwilioVerify/Sources/Networking/NetworkErrorTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  NetworkAdapterTests.swift
+//  NetworkErrorTests
+//
+//  Copyright © 2022 Twilio.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+import XCTest
+@testable import TwilioVerifySDK
+
+class NetworkErrorTests: XCTestCase {
+  func testDataConversion_toServerError_shouldSucceeded() {
```

Comment:
Would be nice to cover the failure flow as well 👍

Reply:
Sorry, I didn't saw that we were testing the codable stuff, I think it would be better to test the logic added to presenter, that will eventually decode that data, and we should cover both flows of that logic added there.

Reply:
Actually I added a failure scenario test. Regarding the Presenter suggestion, that would be nice but currently we don't have any test created for the Demo app. I'll save it for the future! thanks.